Frontend/src/Components/CreateListing/CreateListing.jsx

```diff
@@ -5,12 +5,18 @@ import PhotoSection from './PhotoSection';
 import DescriptionSection from './DescriptionSection';
 import InfoSection from './InfoSection';
 
+const backendApi = import.meta.env.VITE_BACKEND_ADDRESS;
+
 const CreateListing = () => {
   const [description, setDescription] = useState('');
```

Comment:
for lines 11-19, assuming these are all parts of the form input, i think it might be nice to combine all of these states into one, so do something like this 

```
const [formInput, setFormInput] = useState({title: '', description: '', category: '' ...}); 
```
then when you modify each field in onChange, you can update each key doing something like this 
```
setFormInput({...formInput, [key]: value}); // in the case of title, key would be 'title', and value would be e.target.value
```

this way we reduce the number of useStates we have, and in line 52 we can directly pass in formInput